Cargo.toml

```diff
@@ -13,7 +13,7 @@ edition = "2018"
 documentation = "https://docs.rs/crate/parsec-client"
 
 [dependencies]
-parsec-interface = "0.18.0"
```

Comment:
You can keep `0.18.0` here, the version of the interface will only update it.s patch number and hence will be picked up with this caret requirement.